### PR TITLE
Adding single repo pages and a check for blocked issues

### PR DESCRIPTION
**Pull request by jwietelmann**

### CODE REVIEW

**Comment by lintomat** [comment]
## Pull Request Checklist for Reviewer

- [ ] Is the code valid?
- [ ] Does the change "do no harm"-- does not break anything if taken alone?
- [ ] Is data migrated correctly? E.g.:
  - Are suitable defaults and initial values set for new columns?
  - Is data from any removed or changed column handled appropriately (e.g. moved to a different location, or converted to the column's new type)?
  - Can the migrations be run against the production database without any data loss?
  - Are any migrations reversible? If a migration is run and then reversed, is any data lost?
- [ ] Is there appropriate authentication & authorization logic (permissions)?
- [ ] Are there tests?

- [ ] Do the changes make use of existing code appropriately? (For example, existing react components or rails controllers/models. Could be via composition, extension or changes to the existing components/class).
- [ ] Were examples added for code that is intended to/likely to be reused? (via Rev.appExample in the case of JS code, or doc block comments in the case of Rails code)

- [ ] Is there appropriate documentation for the changes? (At least set up instructions for any new dependencies or third party integrations)
- [ ] Does the code match our style guides for the language (i.e. lints are addressed)?
- [ ] Are there appropriate comments?

- [ ] If change introduces annotations like HACK, TODO or FIXME, has a plan been established for fixing the HACK/FIXME in a timely manner. Have appropriate follow-up issues been logged?

- [ ] For PRs that introduce front-end changes-- are there desktop and mobile screenshots attached?

File: app/controllers/checks_controller.rb [hunk 1]
```diff
@@ -3,4 +3,9 @@ class ChecksController < ApplicationController
   def index
     @checks = Checkers.check_all!
   end
+
```
Comment (lintomat) [inline app/controllers/checks_controller.rb]: Expected changes or additions to a test file called [checks_controller_spec.rb](https://github.com/revelrylabs/slack-agile/blob/5820c571ce4876c563cd50f0c8aaefeb1e10cc98/spec/controllers/checks_controller_spec.rb)

File: app/models/checkers/blocked.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module Checkers
```
Comment (lintomat) [inline app/models/checkers/blocked.rb]: Expected changes or additions to a test file called [blocked_spec.rb](https://github.com/revelrylabs/slack-agile/blob/5820c571ce4876c563cd50f0c8aaefeb1e10cc98/spec/models/checkers/blocked_spec.rb)

File: app/models/checkers/right_tags.rb [hunk 3]
```diff
@@ -8,6 +8,7 @@ class RightTags < Base
       'in review',
       'qa',
       'uat',
+      'blocked',
```
Comment (lintomat) [inline app/models/checkers/right_tags.rb]: Expected changes or additions to a test file called [right_tags_spec.rb](https://github.com/revelrylabs/slack-agile/blob/5820c571ce4876c563cd50f0c8aaefeb1e10cc98/spec/models/checkers/right_tags_spec.rb)

File: app/models/repo.rb [hunk 4]
```diff
@@ -6,6 +6,10 @@ def self.from_gh(gh)
     new(org: gh['owner']['login'], repo: gh.name)
   end
 
+  def self.from_name_with_default_org(name)
```
Comment (lintomat) [inline app/models/repo.rb]: Expected changes or additions to a test file called [repo_spec.rb](https://github.com/revelrylabs/slack-agile/blob/5820c571ce4876c563cd50f0c8aaefeb1e10cc98/spec/models/repo_spec.rb)

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.herokuapp.com/relint/revelrylabs/slack-agile/16).

**Comment by prehnRA** [comment]
:shipit:

**Pull request merged**